bigbluebutton-html5/imports/ui/components/fullscreen-button/component.jsx

```diff
@@ -72,18 +73,22 @@ const FullscreenButtonComponent = ({
   });
 
   return (
-    <div className={wrapperClassName}>
-      <Button
-        color="default"
-        icon={!isFullscreen ? 'fullscreen' : 'exit_fullscreen'}
-        size="sm"
-        onClick={() => handleToggleFullScreen(fullscreenRef)}
-        label={formattedLabel(isFullscreen)}
-        hideLabel
-        className={cx(styles.button, styles.fullScreenButton, className)}
-        data-test="presentationFullscreenButton"
-      />
-    </div>
+    <>
+      {isFullscreen && <PollingContainer />}
```

Comment:
I'm not sure this is the best place to render the `PollingContainer`. It might make more sense to render this under the parent element that is being set to full-screen instead of the `FullscreenButtonComponent`

Reply:
moved `pollingContainer` to `PresentationArea`, and removed the duplicated component when in fullscreen